pynetdicom/fsm.py

```diff
@@ -166,13 +170,18 @@ def AE_1(dul: "DULServiceProvider") -> str:
     str
         ``'Sta4'``, the next state of the state machine.
     """
+    # A-ASSOCIATE (request) primitive received from local user
+    primitive = cast("A_ASSOCIATE", dul.to_provider_queue.get(False))
```

Comment:
This needs to be taken off the `to_provider_queue` otherwise it just keeps emitting Evt1